components/apimgt/org.wso2.carbon.apimgt.hostobjects/src/main/java/org/wso2/carbon/apimgt/hostobjects/APIStoreHostObject.java

```diff
@@ -2543,6 +2586,40 @@ public static SubscriptionResponse jsFunction_addAPISubscription(Context cx, Scr
         return addSubscriptionResponse;
     }
 
+    public static SubscriptionResponse jsFunction_addAPISubscriptionByAppId(Context cx, Scriptable thisObj,
+            Object[] args, Function funObj)
+            throws APIManagementException {
```

Comment:
merge lines 2590 and 2591

Reply:
Fixed with 89b6956b5c1641bd96ee670c827bbf6d0a87550a